sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcSourceSuite.scala

```diff
@@ -644,6 +645,28 @@ class OrcSourceSuite extends OrcSuite with SharedSparkSession {
     }
   }
 
+  test("SPARK-37728: Reading nested columns with ORC vectorized reader should not " +
```

Comment:
What I meant was that this should be in `OrcQuerySuite` instead of `OrcSourceSuite` to provide a test coverage for v1 and v2.

Reply:
This PR is a fix for [SPARK-34862]. The unit test "SPARK-34862: Support ORC vectorized reader for nested column" is in OrcSourceSuite.scala on branch-3.2. That's why I put my unit test in OrcSourceSuite.scala. Do you think I should put it in OrcQuerySuite.scala?

Reply:
> What I meant was that this should be in `OrcQuerySuite` instead of `OrcSourceSuite` to provide a test coverage for v1 and v2.
This PR is a fix for [SPARK-34862]. The unit test "SPARK-34862: Support ORC vectorized reader for nested column" is in OrcSourceSuite.scala on branch-3.2. That's why I put my unit test in OrcSourceSuite.scala. Do you think I should put it in OrcQuerySuite.scala?

Reply:
> What I meant was that this should be in `OrcQuerySuite` instead of `OrcSourceSuite` to provide a test coverage for v1 and v2.

This PR is a fix for [SPARK-34862]. The unit test "SPARK-34862: Support ORC vectorized reader for nested column" is in OrcSourceSuite.scala on branch-3.2. That's why I put my unit test in OrcSourceSuite.scala. Do you think I should put it in OrcQuerySuite.scala?

Reply:
- First of all, this PR is a fix for SPARK-37728. Please don't make it mix with the other JIRA although that is the root cause of your JIRA.
- Second, no. You are ignoring my comment about test coverage, `Please recover the test coverage like the original PR`.

Reply:
> * First of all, this PR is a fix for [SPARK-37728](https://issues.apache.org/jira/browse/SPARK-37728). Please don't make it mix with the other JIRA although that is the root cause of your JIRA.
> * Second, yes. It's for my previous comment, "Please recover the test coverage like the original PR". Do you have another reason to remove a test coverage?

OK. Now I moved this unit test case to OrcQuerySuite.scala.

Reply:
Yeah I confirm we need to create unit test in `OrcQuerySuite.scala` instead. Just for context, I moved the unit test from `OrcSourceSuite` to `OrcQuerySuite` when adding support for DSv2 - https://github.com/apache/spark/pull/33842 . Thanks for @dongjoon-hyun pointing it out.